Doc/library/asyncio-stream.rst

```diff
@@ -226,23 +226,48 @@ StreamWriter
    directly; use :func:`open_connection` and :func:`start_server`
    instead.
 
-   .. coroutinemethod:: awrite(data)
+   .. method:: write(data)
 
       Write *data* to the stream.
 
-      The method respects flow control, execution is paused if the write
-      buffer reaches the high watermark.
+      If the method is called using ``await stream.write(data)`` syntax it
```

Comment:
I'd replace with the following:

```
The method attempts to write the data to the underlying socket immediately.
If that fails, the data is queued in an internal write buffer until it can be
sent.

Starting with Python 3.8, it is possible to directly await on the `write()` method::

     await stream.write(data)

The "await" pauses the current coroutine until the data is written to the socket.

Below is an equivalent code that works with Python <= 3.7::

    stream.write(data)
    await stream.drain()
```

Reply:
Done